src/components/SidePanel/NavGroup/FilterNavGroup/FilterGroup/index.tsx

```diff
@@ -65,154 +69,175 @@ const FilterGroup: React.FC<FilterGroupProps> = ({
       ? "Filter cannot be empty"
       : `Invalid Regular Expression: ${getRegexpError(newFilterName)}`;
 
+  const iconTooltip =
+    !isValid && !isEditing ? (
+      <IconWithTooltip color={red.base} glyph="ImportantWithCircle">
+        Invalid filter expression, please update it!
+        <Error>{validationMessage}</Error>
+      </IconWithTooltip>
+    ) : null;
```

Comment:
[super nit] My preference would be to put the conditionals outside of the `iconTooltip` var itself